easybuild/easyconfigs/n/NanoComp/NanoComp-1.10.1-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,39 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonPackage'
+
+name = 'NanoComp'
+version = '1.10.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/wdecoster/NanoComp'
+description = "Comparing runs of Oxford Nanopore sequencing data and alignments"
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['552334d8048165374a3c3090ff75cb8177ee2e518770b53aeedf7fc9a6d6bc07']
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('Pysam', '0.15.3'),
+    ('nanomath', '0.23.1', versionsuffix),
+    ('nanoget', '1.12.1', versionsuffix),
+    ('NanoPlot', '1.28.4', versionsuffix),
+    ('plotly.py', '4.4.1'),
+    ('orca', '1.3.0'),
+    ('joypy', '0.2.2', versionsuffix),
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+sanity_check_commands = [
+    "NanoComp --help",
```

Comment:
it's fine to collapse this to a single line imho:

```python
sanity_check_commands = ["NanoComp --help"]
```

likewise in the other easyconfigs